static/js/components/dashboard/FinalExamCard.js

```diff
@@ -151,11 +151,27 @@ const renderPearsonTOSDialog = (open, show, submitPearsonSSO, pearson) => (
     autoScrollBodyContent={true}>
     <div className="dialog-container">
       <img src="/static/images/pearson_vue.png" width="180"/>
-      <h3 className="dialog-title">Test Registration is completed on the <br/>Pearson VUE website</h3>
-      <p>
-        I acknowledge that by clicking Continue I will be leaving the MicroMasters <br/>website and going to
-        the Pearson VUE website, and that I accept the <br/>Pearson VUE Group’s <a target="_blank"
-          href="https://home.pearsonvue.com/Legal/Privacy-and-cookies-policy.aspx">Terms of Service</a>.
+      <h3 className="dialog-title">You are being redirected to Pearson VUE’s website.</h3>
+      <p className="tos-container">
```

Comment:
could we make this container a `<div>` and then, inside of that div, use `<p>` tags instead of `<br/>` all over the place to delineate paragraphs? I think that would result in cleaner HTML.